.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -4,5 +4,5 @@ contact_links:
     url: https://join.slack.com/t/infectionmonkey/shared_invite/enQtNDU5MjAxMjg1MjU1LWM0NjVmNWE2ZTMzYzAxOWJiYmMxMzU0NWU3NmUxYjcyNjk0YWY2MDkwODk4NGMyNDU4NzA4MDljOWNmZWViNDU
     about: Our community Slack channel - you can ask questions or suggest things here.
   - name: FAQs
-    url: https://www.guardicore.com/infectionmonkey/faq/
+    url: https://www.guardicore.com/infectionmonkey/docs/faq/
```

Comment:
Should link to Akamai.

Reply:
When we host our documentation under different domain then we will need to change all documentation links. Our documentation is still hosted under Guardicore.